assets/js/Containers/Databrowser/index.js

```diff
@@ -72,6 +72,7 @@ class Databrowser extends React.Component {
           grid_label: window.grid_label,
           time_aggregation: window.time_aggregation,
           fs_type: window.fs_type,
+          cmor_table: window.cmor_table,
```

Comment:
Do we really need that? @Karinon ?

Reply:
You tell me. If the CMOR-Table is never visible, then there is no reason to add that.

Reply:
It's visible.

Reply:
This is necessary so the metadata of each facet can be visible (and it has to be explicitly written one by one, apparently), without the line cmor_table could not look upon the metadata.js file.

Reply:
I'd rather favour a solution where this can be done dynamically instead of always tempering with the code if we decide on displaying more information.

Reply:
I agree, unfortunatelly I have no idea yet. After I answered I was asking chatgpt how to do it...